Lux/lux_react/src/components/TabNavigation.jsx

```diff
@@ -6,8 +6,11 @@ import { FaBullseye } from 'react-icons/fa';
 import { BsBrush } from 'react-icons/bs';
```

Comment:
Tab navigation is scrolling left to right - put icons closer together so this doesn't happen on mobile.